src/FixedDataTableCellGroup.js

```diff
@@ -82,7 +82,7 @@ var FixedDataTableCellGroupImpl = React.createClass({
       if (!recycable || (
             currentPosition - props.left <= props.width &&
             currentPosition - props.left + columnProps.width >= 0)) {
-        var key = 'cell_' + i;
+        var key = columnProps.columnKey || 'cell_' + i;
```

Comment:
Naw, I was afraid we were breaking backwards compatibility.  But if we already have a documented requirement that column keys are unique, then no need.